invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordCitationField.js

```diff
@@ -86,8 +86,9 @@ export class RecordCitationField extends Component {
         this.setState({
           loading: false,
           citation: "",
-          error:
-          i18next.t('An error ocurred while generating the citation, please try again. If this error continues please contact our technical support.'),
+          error: i18next.t(
+            "An error ocurred while generating the citation, please try again. If this error continues please contact our technical support."
```

Comment:
Perhaps we should just replace this with ``An error occurred while generating the citation.`` I mean, it probably doens't help trying again, and there's no information on how to contact the support.

There's also a typo:
```suggestion
            "An error occurred while generating the citation, please try again. If this error continues please contact our technical support."
```